Podcast/RecommendedEpisodesOuterTableViewCell.swift

```diff
@@ -37,6 +38,7 @@ class RecommendedEpisodesOuterTableViewCell: UITableViewCell, UITableViewDelegat
         tableView.delegate = self
         tableView.dataSource = self
         tableView.register(EpisodeTableViewCell.self, forCellReuseIdentifier: "Cell")
+        tableView.register(NullProfileTableViewCell.self, forCellReuseIdentifier: "NullCell")
```

Comment:
we don't need to register a `reuseIdentifier` because we never reuse this cell so delete this line